### ISSUE TITLE
There is a bug in the reconnect logic

### ISSUE THREAD

**Issue by atleta** [cytopia/smtp-user-enum#16]
After causing a network failure (sending my laptop to sleep, effectively disconnecting the wifi adapter) I got the following exception:

```
Traceback (most recent call last):
  File "env/bin/smtp-user-enum", line 571, in <module>
    main()
  File "env/bin/smtp-user-enum", line 548, in main
    enum_users(
  File "env/bin/smtp-user-enum", line 155, in enum_users
    s = init_connection(
TypeError: init_connection() missing 1 required positional argument: 'verbose'
```

Looking into the code, there is a bug in enum_users where it makes a call to `init_connection`. It happens in two places: line 129 and line 155. You pass 7 parameters but `init_connection` requires 8. Based on the argument names, the `helo` is the missing one and it's not even being passed to `enum_users` (in `conn`).

The call in the `main` function doesn't have this problem (otherwise the app would fail right away).

I can create a PR as I have fixed it for myself locally already. (Though the fix is just a few characters, really.)

**Comment by cytopia** [comment]
@atleta thanks for the info. I've created a PR here: https://github.com/cytopia/smtp-user-enum/pull/17

**Comment by cytopia** [comment]
Fixed and made available as `0.7.0`
